scripts/cibuild

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -e
+
+if [[ -n "${RF_DEBUG}" ]]; then
+    set -x
+fi
+
+DIR="$(dirname "$0")"
+
+function usage() {
+    echo -n \
+         "Usage: $(basename "$0")
+Build application for staging or a release.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+      if [[ -n "${RF_DOCS_BUCKET}" ]]; then
+          echo "Building docs container"
+          docker-compose build docs
+
+          # echo "Running tests"
+          # "${DIR}/test"
```

Comment:
We should remove these lines. They can be added back as part of https://github.com/azavea/raster-foundry/issues/1472.